### PR TITLE
Fixed 'No' deployment response not updating config files

### PR DESCRIPTION
**Pull request by ggotti**
- Added flags to allow build/image upload to be skipped.
- Saying 'N' to the deployment option will now still output the files.

### CODE REVIEW

File: generators/more-aws/index.js [hunk 1]
```diff
@@ -48,6 +68,11 @@ module.exports = class extends BaseGenerator {
                 this.packageName = this.config.get('packageName');
                 this.packageFolder = this.config.get('packageFolder');
                 this.prodDatabaseType = this.config.get('prodDatabaseType');
+                this.hasAWSConfig = this.config.get('aws') ? true : false;
+                this.deployNow = this.options['skip-install'];
+                this.skipUpload = this.options['skip-upload'];
+                this.skipBuild = this.options['skip-build'];
```
Comment (carvallegro) [inline generators/more-aws/index.js]: Could you put the options fetching line in an `option()` method before `bonjour()`?

Comment (carvallegro) [reply generators/more-aws/index.js]: Also, could you rename `bonjour` into `configMining`or something? `bonjour` was used as a one liner to display the current step

File: generators/more-aws/prompts.js [hunk 2]
```diff
@@ -360,11 +360,8 @@ function askDeployNow() {
...
     return this.prompt(prompts).then((props) => {
         const deployNow = props.deployNow;
-        if (deployNow) {
-            done();
-        } else {
-            this.abort = true;
-            done();
-        }
+
+        this.deployNow = deployNow;
```
Comment (carvallegro) [inline generators/more-aws/prompts.js]: `this.deployNow = props.deployNow;` ?

File: generators/more-aws/index.js (hunk 1 above)
Comment (ggotti) [reply generators/more-aws/index.js]: 🌮

File: generators/more-aws/prompts.js (hunk 2 above)
Comment (ggotti) [reply generators/more-aws/prompts.js]: 🌮

**Pull request merged**